packages/stripe-integration/src/stripe-upe/stripe-ocs-payment-strategy.ts

```diff
@@ -0,0 +1,413 @@
+            clientSecret: paymentMethod.clientToken,
+            locale: formatLocale(shopperLanguage),
+            appearance: this._getElementAppearance(style),
+            fonts: [
+                {
+                    cssSrc: 'https://fonts.googleapis.com/css?family=Montserrat:700,500,400%7CKarla:400&display=swap', // TODO: get style from theme
+                },
+            ],
+        });
+
+        const { getBillingAddress, getShippingAddress } = state;
+        const { postalCode } = getShippingAddress() || getBillingAddress() || {};
+
+        const stripeElement: StripeElement =
+            this.stripeElements.getElement(StripeElementType.PAYMENT) ||
+            this.stripeElements.create(StripeElementType.PAYMENT, {
+                fields: {
+                    billingDetails: {
+                        email: StripeStringConstants.NEVER,
+                        address: {
+                            country: StripeStringConstants.NEVER,
+                            city: StripeStringConstants.NEVER,
+                            postalCode: postalCode
+                                ? StripeStringConstants.NEVER
+                                : StripeStringConstants.AUTO,
+                        },
+                    },
+                },
+                wallets: {
+                    applePay: StripeStringConstants.NEVER,
+                    googlePay: StripeStringConstants.NEVER,
+                },
+                layout: {
+                    type: 'accordion',
+                    defaultCollapsed: false,
+                    radios: true,
+                    spacedAccordionItems: false,
+                    visibleAccordionItemsCount: 0,
+                },
+            });
+
+        this.stripeUPEIntegrationService.mountElement(stripeElement, containerId);
+
+        stripeElement.on('ready', () => {
+            render();
+        });
+
+        stripeElement.on('change', (event: StripeEventType) => {
+            this._onStripeElementChange(event, gatewayId, methodId, paymentMethodSelect);
+        });
+
+        handleClosePaymentMethod?.(this._collapseStripeElement.bind(this));
+    }
+
+    private async _loadStripeJs(
+        stripePublishableKey: string,
+        stripeConnectedAccount: string,
+    ): Promise<StripeUPEClient> {
+        if (this.stripeUPEClient) {
+            return this.stripeUPEClient;
+        }
+
+        return this.scriptLoader.getStripeClient(stripePublishableKey, stripeConnectedAccount);
+    }
+
+    private _getElementAppearance(
+        style?: StripeUPEPaymentInitializeOptions['style'],
+    ): StripeUPEAppearanceOptions | undefined {
+        if (!style) {
+            return;
+        }
+
+        const titleFontSize = '15px'; // TODO: get style from theme
+        const titleFontWeight = '700'; // TODO: get style from theme
+        const titleColor = '#5f5f5f'; // TODO: get style from theme
```

Comment:
Are you sure that we can release it with hardcoded styles?
Is it hidden by experiment?

Reply:
It is not a release, only a first PR with strategy creation.
Yes, it is hidden by experiment and will be released in the next year.
This hardcoded values it is only a mock, and will be changed in future tasks and PRs.